### ISSUE TITLE
v2.6.2 did not bump version number

### ISSUE THREAD

**Issue by jeremysherriff** [linuxserver/ifai#1389]
v2.6.2 released earlier today did not bump the version string in the app to `2.6.2`, web app still shows `2.6.1`.
Screenshot from app:
![image](https://github.com/user-attachments/assets/4ca48e01-783a-40d2-b518-47b9ff461b9b)

Output from `docker inspect`:
```
root@stuff:~# docker inspect b71609f93409
[
    {
        "Id": "sha256:b71609f93409df59c08b38fe4cfdf3118a0f0c1dfa8ef48e5afa93849e33c445",
        "RepoTags": [
            "lscr.io/linuxserver/heimdall:latest"
        ],
...
        "Created": "2024-11-04T10:48:20.050492974Z",
...
            "Labels": {
                "build_version": "Linuxserver.io version:- v2.6.2-ls287 Build-date:- 2024-11-04T10:47:09+00:00",
```

The world's smallest issue :)
I won't make a pull request as the fix would require a release bump, changing the version number again...

**Comment by KodeStar** [comment]
Whoops

**Comment by KodeStar** [comment]
Should be fixed in 2.6.3
